MREUnityRuntime/MREUnityRuntimeLib/Constants.cs

```diff
@@ -54,5 +54,12 @@ static Constants()
 		 * this *is* a big deal, and requires discussion and signoff from the dev team.
 		 */
 		internal const string MinimumSupportedSDKVersion = "0.13";
+
+		/*
+		 * Enable physics bridge
+		 * Work in progress feature to improve shared physics synchronization.
+		 * It is NOT compatibile with SDK versions older than 0.19.
+		 */
+		internal const bool UsePhysicsBridge = true;
```

Comment:
I tried to use an automated expression, like string.compare with the version as an input but the compiler did not accept it

Reply:
we'll figure it out by the time we enable physics bridge by default

---
In reply to: [441634208](https://github.com/microsoft/mixed-reality-extension-unity/pull/188#discussion_r441634208) [](ancestors = 441634208)